### PR TITLE
Update list to 6.0

### PR DESCRIPTION
**Pull request by cosminsandu**
Update list to 6.0

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -5,177 +5,186 @@ permalink: /
...
+- [ ] [Data Validation](/topics/data-validation.md)
+    - [ ] [PHP object validation](/topics/data-validation.md#php-object-validation)
+    - [ ] [Built-in validation constraints](/topics/data-validation.md#built-in-validation-constraints)
+    - [ ] [Validation scopes](/topics/data-validation.md#validation-scopes)
+    - [ ] [Validation groups](/topics/data-validation.md#validation-groups)
+    - [ ] [Group sequence](/topics/data-validation.md#group-sequence)
+    - [ ] [Custom callback validators](/topics/data-validation.md#custom-callback-validators)
+    - [ ] [Violations builder](/topics/data-validation.md#violations-builder)
+- [ ] [Dependency Injection](/topics/dependency-injection.md)
+    - [ ] [S[ ] ervice container](/topics/dependency-injection.md#service-container)
```
Comment (ThomasBerends) [inline readme.md]: Small typo

**Comment by ThomasBerends** [comment]
@cosminsandu This is absolutely amazing! Thank you a lot!

I found a small typo, but that's the only thing. Could you fix that? Then I'll merge it right away.

Thank you again!

File: readme.md (hunk 1 above)
Comment (cosminsandu) [reply readme.md]: was fixed in last pull commit

**Comment by ThomasBerends** [comment]
@cosminsandu Merged! Thank you for your massive contribution!

**Pull request merged**